Runtime/Behaviours/CinemachineVirtualCamera.cs

```diff
@@ -470,7 +474,18 @@ private CameraState CalculateNewState(Vector3 worldUp, float deltaTime)
             LookAtTargetAttachment = 1;
 
             // Initialize the camera state, in case the game object got moved in the editor
-            CameraState state = PullStateFromVirtualCamera(worldUp, ref m_Lens);
+            if (m_Lens.ModeOverride == LensSettings.OverrideModes.InheritFromCamera &&
+                m_OverrideModeCache == LensSettings.OverrideModes.InheritFromCamera)
+            {
+                m_LensCache = m_Lens; // save camera lens state
+            }
+            CameraState state = PullStateFromVirtualCamera(Vector3.up, ref m_Lens);
+            if (m_Lens.ModeOverride == LensSettings.OverrideModes.InheritFromCamera &&
+                m_OverrideModeCache != LensSettings.OverrideModes.InheritFromCamera)
+            {
+                m_State.Lens = m_Lens = m_LensCache; // restore original camera lens state
+            }
+            m_OverrideModeCache = m_Lens.ModeOverride;
```

Comment:
This is too naive.  So many ways to break this.
The only way to implement overrides properly is to have a stack in the brain.